docs/docs/non-cloud-deployment.md

```diff
@@ -0,0 +1,115 @@
+* Ubuntu 20.04 LTS Server 64bits
+* Red Hat Enterprise Linux Server release 7.6 64bits
+* Red Hat Enterprise Linux Server release 8.2 64bits
+* CentOS 8.2 64bits
+* Fedora 31 Server 64bits
+
+
+## Install the kernel driver and system software
+
+You can get the installers for the system software from Intel
+[here](https://01.org/intel-software-guard-extensions/downloads).
+
+Follow this
+[user guide](https://download.01.org/intel-sgx/sgx-linux/2.13.3/docs/Intel_SGX_Installation_Guide_Linux_2.13.3_Open_Source.pdf)
+to install the driver and the system software.
+
+Intel provides:
+
+* APT repositories for Ubuntu
+* Cross-distro installer binaries for other platforms, which set up the system software and install the kernel driver.
+
+!!!Important
+
+    You need to re-run the installer after upgrading the kernel.
+
+Alternatively, you can [compile the system software](https://github.com/intel/linux-sgx/releases/tag/sgx_2.13.3)
+yourself.
+The [kernel driver is also available on GitHub](https://github.com/intel/linux-sgx-driver).
+
+A daemon `aesmd` helps to implement SGX remote attestation and machine provisioning. This daemon comes as part of the 
+SGX platform services software and is set up during installation.
+
+A quick summary:
+
+1. Download and run the driver installer binary (all distros).
+2. For Ubuntu users:
+    * For Ubuntu 18 LTS: `echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' > /etc/apt/sources.list.d/intelsgx.list`
+    * For Ubuntu 20 LTS: `echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intelsgx.list`
+    * Add the Intel package signing key: `wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -`
+    * Run `apt-get update`.
+    * Run `apt-get install libssl-dev libcurl4-openssl-dev libprotobuf-dev libsgx-urts libsgx-launch libsgx-epid libsgx-quote-ex`
+3. For non-Ubuntu users, use the SDK installer (which also installs the platform services software).
+
+These steps will start the `aesm_service`.
+
+
+## Limited network connectivity
+
+The enclave host machine needs to contact Intel's attestation servers to prove to third parties that it's a genuine, 
+unrevoked CPU running with the latest secure configuration. So, if the machine has limited connectivity, you must use 
+an outbound HTTP[S] proxy server.
+
+You need to set up your proxy settings in the `/etc/aesmd.conf` file inside the `aesmd` service.
+
+The program that uses Conclave will also need to make web requests to https://api.trustedservices.intel.com. So you
+need to [provide Java with HTTP proxy settings](https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html)
+as well.
+
+## Using containers
+
+To configure Docker for use with SGX, you must pass at least these flags when creating the container:
+
+`--device=/dev/isgx -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket`
```

Comment:
I believe that this is only for EPID and for out of kernel driver. I.e. it is quite old piece of documentation.
Maybe it can be part of the refactoring that we discussed offline in the channel?